src/ishikawa_backend.erl

```diff
@@ -134,75 +136,46 @@ init([Fun]) ->
     {ok, Members} = ?PEER_SERVICE:members(),
     lager:info("Initial membership: ~p", [Members]),
 
-    {ok, #state{actor=Actor, vv=VClock, members=Members, svv=SVV, rtm=RTM, to_be_delivered_queue=ToBeDeliveredQueue, to_be_ack_queue=ToBeAckQueue, msg_handling_fun=MessageHandlingFun}}.
+    {_, TRef} = timer:send_interval(?WAIT_TIME_BEFORE_CHECK_RESEND, check_resend),
```

Comment:
One thought: instead of `send_interval`, you could do a single timer `send_after` and then reschedule it in the `check_resend` handler: this is mainly because if processing gets to be *slower* than the interval, you don't want them to back up, so you should only reschedule once you've finished processing.